lib/aescrypt.rb

```diff
@@ -74,10 +84,23 @@ def self.decrypt_data(encrypted_data, key, iv, cipher_type)
   #:arg: iv => String
   #:arg: cipher_type => String  
   def self.encrypt_data(data, key, iv, cipher_type)
-    aes = OpenSSL::Cipher::Cipher.new(cipher_type)
+    self.warn_if_ecb cipher_type
+
+    aes = OpenSSL::Cipher.new(cipher_type)
```

Comment:
Technically, `aes` is not necessarily the right variable name, as `cipher_type` might specify non-AES.